app.py

```diff
@@ -121,6 +126,13 @@ def cli(ctx=None, verbose=False, pypi_rss_feed=None, graph_hosts=None, graph_por
         only_if_package_seen=only_if_package_seen
     )
 
+    push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY')
+    if push_gateway:
+        try:
+            push_to_gateway(push_gateway, job='package-releases', registry=prometheus_registry)
+        except Exception as e:
+            print('An error occurred pushing the metrics: {}'.format(str(e)))
```

Comment:
The casting is not that necessary, the previous suggestion was to use _LOGGER instead of raw `print`. Once we will do log aggregation, the `thoth-common` package handles all the logging logic so we will capture such errors and propagate them where needed (see recent rsyslog endpoint discussion).